### PR TITLE
uniphier: fix section of  ROTPK hash

### PR DESCRIPTION
**Pull request by masahir0y**

### CODE REVIEW

File: plat/socionext/uniphier/uniphier_rotpk.S [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 	.global uniphier_rotpk_hash
 	.global uniphier_rotpk_hash_end
+	.section .rodata.uniphier_rotpk_hash, "aS"
```
Comment (davidcunado-arm) [inline plat/socionext/uniphier/uniphier_rotpk.S]: Should this be "a" and not "aS"?

**Comment by davidcunado-arm** [comment]
jenkins: test this please

File: plat/socionext/uniphier/uniphier_rotpk.S (hunk 1 above)
Comment (masahir0y) [reply plat/socionext/uniphier/uniphier_rotpk.S]: You are right.
Looks like I copy-pasted a wrong one.

I googled "S", and it says "section contains zero terminated strings".
This section just contains raw data.

I fixed it and pushed.  Thanks!

**Comment by masahir0y** [comment]
I updated this PR as David.

**Comment by davidcunado-arm** [comment]
jenkins: test this please

**Comment by davidcunado-arm** [comment]
jenkins: test this please

**Pull request merged**